robot/description/launch/caffeine_gazebo.launch

```diff
@@ -52,8 +52,14 @@
       <arg name="camera_ns" value="zed/zed_node"/>
    </include>
 
+   <!-- Set up robot_localization odometry -->
```

Comment:
This comment has a weird ring to it, I would rather go with `<!-- Compute Odometry from Sensor Fusion -->`. But since we are going to merge with the NavSat code, we can just take that into account